src/components/og-spinner/og-spinner.scss

```diff
@@ -0,0 +1,102 @@
+// =============================================================================
+// CSS CUSTOM PROPERTIES
+// =============================================================================
+
+// -----------------------------------------------------------------------------
+// Base
+// -----------------------------------------------------------------------------
+
+:host {
+    // All Component Variables
+    // Map Variables to Properties
```

Comment:
Maybe just check the comments regarding the use of capital letters and an ending dot. Does not seem consistent.